src/status_im/contexts/chat/home/view.cljs

```diff
@@ -13,13 +14,32 @@
     [status-im.common.home.empty-state.view :as common.empty-state]
     [status-im.common.home.header-spacing.view :as common.header-spacing]
     [status-im.common.resources :as resources]
+    [status-im.config :as config]
     [status-im.contexts.chat.actions.view :as chat.actions.view]
     [status-im.contexts.chat.home.chat-list-item.view :as chat-list-item]
     [status-im.contexts.chat.home.contact-request.view :as contact-request]
     [status-im.contexts.shell.jump-to.constants :as jump-to.constants]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn- f-chat-open-indicator
+  []
+  (let [current-chat-id  (rf/sub [:chats/current-chat-id])
+        view-id          (rf/sub [:view-id])
+        background-color (reanimated/use-shared-value "transparent")]
+    (if (and (= view-id :chats-stack) current-chat-id)
+      (reanimated/animate-delay background-color "red" 300)
+      (reanimated/set-shared-value background-color "transparent"))
+    [reanimated/view
```

Comment:
imo it would be nicer to have some text or something more descriptive here. It's likely if other devs see this reb box they will be confused of what is happening and we are creating noise that is easily avoidable. 👍

Reply:
yeah, plan was to link code with PR. (chicken egg problem)
forget to add comment, after pushing. Thank you for reminding.

Reply:
ah I didn't mean in the code tbh, I mean in the ui

Reply:
we could have a more generic mechanism, like a floating button with an error/warning symbol that  shows more info once pressed

Reply:
More text might create interference with testing and development.
Once PR is approved I have plan to ping qa/devs in discord about this information.
(I can also add in some doc, but probably it will get unnoticed)

Reply:
> we could have a more generic mechanism, like a floating button with an error/warning symbol that shows more info once pressed

this is interesting idea. Once implemented this way of indicating/communicating bug in PR builds can be used at other places too.

Reply:
Oh we were supposed to implement something like that for documentation. (Don't know where we are on that)

Reply:
- Create a view in shell screen (with high z index)
- Create a mechanism to track different types of errors like here
- That return a string of error
- when string is not null a button is shown in right corner of status bar (so it don't interfere with app flow)
- when button pressed so popup with that error.


Just a rough idea, wdyt @J-Son89

Reply:
yeah for me sounds great, maybe we can check in the mobile channel to see if anyone else has any suggestions/thoughts about it! :)

Reply:
Updated the code. How's this looks?

<details>
  <summary> Video </summary>
 
<div align="center">

https://github.com/status-im/status-mobile/assets/17097240/5794b4f0-96f8-4fae-bb4c-dd5013447ad6

</div>

</details>

Reply:
nice @Parveshdhull! yes this looks great to me! Very clear to see what's happening now :)